src/hooks/use-menu/__tests__/index.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { renderHook } from '@testing-library/react-hooks'
+import { useMenu } from '..'
+
+describe('useMenu', () => {
+  it('getTriggerProps should apply aria and class attributes correctly', () => {
+    const { result } = renderHook(() => useMenu())
+    const triggerProps = result.current.getTriggerProps({
+      className: 'custom-class',
+    })
+
+    expect(triggerProps['aria-haspopup']).toBe(true)
+    expect(triggerProps['aria-expanded']).toBe(false)
```

Comment:
**todo:** the presence of the `className` attribute isn't being asserted here, but the test description suggests we care about it. We either need to add the assertion or update the test description.